public/app/features/profile/state/reducers.ts

```diff
@@ -78,7 +78,7 @@ export const slice = createSlice({
         id: session.id,
         isActive: session.isActive,
         seenAt: dateTimeFormatTimeAgo(session.seenAt),
-        createdAt: dateTimeFormat(session.createdAt, { format: 'MMMM DD, YYYY' }),
+        createdAt: new Date(session.createdAt),
```

Comment:
This is possibly the one contentious part of this PR that I want feedback/extra eyes on. 

To format the date correctly, with respect to preferred localisation, in the view, we need access to the createdAt as an actual Date object. Besides, I think it is "less correct" to store the date as a formatted string in the store anyway.

Reply:
mmmmmm i agree this is a little contentious 😛 

storing objects in a redux store is a little funky. i think you're only supposed to store JSON serializable stuff in there? could we store either an ISO timestamp or the UNIX timestamp and creating the Date object at runtime? 🤔

Reply:
Good shout. I didn't know storing non-json serialisable values in Redux was not recommended!